src/core/core-varlink.c

```diff
@@ -45,6 +46,42 @@ static bool user_match_lookup_parameters(LookupParameters *p, const char *name,
         return true;
 }
 
+static int vl_method_get_oomd_enabled_cg_paths(Varlink *link, JsonVariant *parameters, VarlinkMethodFlags flags, void *userdata) {
+        _cleanup_(json_variant_unrefp) JsonVariant *v = NULL;
+        _cleanup_strv_free_ char **p = NULL;
+        Manager *m = userdata;
+        CGroupContext *c;
+        Unit *u;
+        int r;
+
+        assert(link);
+        assert(m);
+
+        if (json_variant_elements(parameters) > 0)
+                return varlink_error_invalid_parameter(link, parameters);
+
+        /* Only slices support EnableOomdKill= right now */
+        LIST_FOREACH(units_by_type, u, m->units_by_type[UNIT_SLICE]) {
+                if (unit_active_state(u) == UNIT_ACTIVE) {
+                        c = unit_get_cgroup_context(u);
+                        if (!c)
+                                continue;
+
+                        if (c->enable_oomd_kill) {
+                                r = strv_extend(&p, u->cgroup_path);
+                                if (r < 0)
+                                        return varlink_error(link, "io.systemd.Oomd.OutOfMemory", NULL);
+                        }
+                }
+        }
+
+        r = json_build(&v, JSON_BUILD_OBJECT(JSON_BUILD_PAIR("cg_paths", JSON_BUILD_STRV(p ?: STRV_MAKE_EMPTY))));
+        if (r < 0)
+                return varlink_error(link, "io.systemd.Oomd.OutOfMemory", NULL);
```

Comment:
this doesn't look like an error to pass client side (in particular as it is not likely to succeed either, since it's going to allocate a json record too. maybe just do "return log_oom()" here.  similar above